plugins/woocommerce/changelog/add-32157_tests_to_disable_welcome_modals

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: add
+
+Add E2E tests to disabled welcome modal #32505
```

Comment:
We will probably also need a changelog in the `admin-e2e-tests` package, although I don't think we have been very strict with that.

Reply:
Sounds good to me! I added this change to the changelog in commit 23b3250.